libraries/Microsoft.Bot.Builder.AI.LUIS/Generator/GeographyV2.cs

```diff
@@ -45,7 +45,9 @@ public GeographyV2(string type, string location)
         /// <summary>
         /// Different types of geographic locations.
         /// </summary>
+#pragma warning disable CA1034 // Nested types should not be visible (we can't change this without breaking binary compat)
         public static class Types
```

Comment:
>ublic static class Typ [](start = 9, length = 22)

This class does need to be public.  We could have exposed it at the top.

Reply:
I explicitly excluded the rule for backward comapt. But we shouldn't use public nested types in the future.
Some more details here: https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1034?view=vs-2019